src/PIL/__init__.py

```diff
@@ -13,6 +13,7 @@
 
 from . import version
 
+# PILLOW_VERSION and VERSION are deprecated and will be removed in Pillow 6.0.0. Use __version__ instead.
 VERSION = '1.1.7'  # PIL Version
 PILLOW_VERSION = version.__version__
```

Comment:
Could you rebase on the current master to see the full picture?

Reply:
Rebased!